tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py37
```

Comment:
[ASK] In pyproject.toml `python = "^3.6"` is declared. Maybe you can test this with py36?

Reply:
Sorry, this is my mistake (this is for test on python 3.6 but failed).
Fied!